cli/src/cmd/forge/test/mod.rs

```diff
@@ -80,6 +80,10 @@ pub struct TestArgs {
     #[clap(long, short, help_heading = "Display options")]
     json: bool,
 
+    /// Exit tests after first failure
```

Comment:
It looks like `clap` processes help messages in docs format. 
Please have a look #4768